src/test/java/seedu/tassist/storage/JsonAdaptedPersonTest.java

```diff
@@ -51,7 +52,7 @@ public void toModelType_invalidName_throwsIllegalValueException() {
                 new JsonAdaptedPerson(INVALID_NAME, VALID_PHONE, VALID_TELE_HANDLE,
                         VALID_EMAIL, VALID_MAT_NUM, VALID_TUT_GROUP, VALID_LAB_GROUP,
                         VALID_FACULTY, VALID_YEAR, VALID_REMARK,
-                        VALID_ATTENDANCE_STRING, VALID_TAGS);
+                        VALID_ATTENDANCE_STRING, VALID_LAB_SCORES_STRING , VALID_TAGS);
```

Comment:
Apply suggestion to everything below
```suggestion
                        VALID_ATTENDANCE_STRING, VALID_LAB_SCORES_STRING, VALID_TAGS);
```